src/App.jsx

```diff
@@ -1,35 +1,13 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] `required`
- [ ] Please enable the disabled eslint because it defeats the purpose of using the linters. 🥂

Reply:
- [ ]  Disable the eslint